src/server/table_definitions.sql

```diff
@@ -0,0 +1,95 @@
+-- a minimal users table, if not using a phpbb3 installation
+-- DROP TABLE IF EXISTS users;
+-- CREATE TABLE users
+-- (
+--     user_id int(10) unsigned NOT NULL AUTO_INCREMENT,
+--     user_type tinyint(2) NOT NULL DEFAULT '0',
+--     username varchar(255) COLLATE utf8_bin NOT NULL DEFAULT '',
+--     user_password varchar(255) COLLATE utf8_bin NOT NULL DEFAULT '',
+--     user_email varchar(100) COLLATE utf8_bin NOT NULL DEFAULT '',
+--     PRIMARY KEY (user_id),
+--     KEY user_type (user_type)
+-- ) ENGINE=InnoDB AUTO_INCREMENT=50 DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
+
+------
+
+-- table used to manage bans issued by wesnothd
+DROP TABLE IF EXISTS ban;
+CREATE TABLE ban
+(
+    ban_userid varchar(100) NOT NULL,
+    ban_end int(10) unsigned NOT NULL DEFAULT '0',
+    ban_ip varchar(100) DEFAULT NULL,
+    ban_email varchar(100) DEFAULT NULL,
+    ban_exclude int(10) unsigned NOT NULL DEFAULT '0',
+    PRIMARY KEY (ban_userid)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+-- extra information as necessary per user
+-- user_lastvisit is used by the phpbb extension displaying the last time the user logged in to the MP server
+-- user_is_moderator determines people who have the abilities granted to MP Moderators
+DROP TABLE IF EXISTS extra;
+CREATE TABLE extra
+(
+    username varchar(100) NOT NULL,
+    user_lastvisit int(10) unsigned NOT NULL DEFAULT '0',
+    user_is_moderator tinyint(4) NOT NULL DEFAULT '0',
+    PRIMARY KEY (username)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+-- information about a single game instance
```

Comment:
Perhaps a bit confusing to say game _instance_ when the columns with INSTANCE in their name refer to wesnothd.